presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -7882,14 +7902,7 @@ public void testQuantifiedComparisonCornerCases(String query)
         List<QueryTemplate.Parameter> lhs = new QueryTemplate.Parameter("lhs").of("1", "NULL");
         List<QueryTemplate.Parameter> relation = new QueryTemplate.Parameter("relation").of("=", "!=", "<", ">", "<=", ">=");
         QueryTemplate queryTemplate = new QueryTemplate("SELECT %lhs% %relation% %quantifier% (%subquery%)");
-        ImmutableList<String> excludedInPredicateQueries = ImmutableList.of(
-                "SELECT NULL != ALL (SELECT * FROM (SELECT 1 WHERE false) as empty_table)",
```

Comment:
removing those lines restored those tests. They were failing because of problems with InPredicate.